main.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_lb_listener" "this" {
   certificate_arn = try(each.value.certificate_arn, null)
 
   dynamic "default_action" {
-    for_each = try([each.value.authenticate_cognito], [])
+    for_each = can(coalesce(each.value.authenticate_cognito)) ? [each.value.authenticate_cognito] : []
```

Comment:
this is not a change that we are going to accept. you should change your variables in your wrapper to make it work with the existing implementation. I would start with not using `null` and instead default to an empty list.

Reply:
> The change is a safer null-safe implementation 

I don't know what is meant by this. but in general, you are applying defaults that conflict with the module - therefore, you should adapt your implementation, not the other way around